.travis.yml

```diff
@@ -5,13 +5,20 @@ rust:
   - stable
   - beta
   - nightly
-install:
-  - '[ "$TRAVIS_RUST_VERSION" == "1.16.0" ] || rustup target add thumbv6m-none-eabi'
+matrix:
+  include:
+    - rust: 1.21.0
+      script:
+        - cargo test --verbose --features kv_unstable
+        - cargo test --verbose --features "kv_unstable std"
+    - rust: stable
```

Comment:
I ended up rolling in this tweak to our CI so we do the embedded build as a separate task like the `kv_unstable` one, but it does mean we only check it on stable instead of also on beta and nightly. I can revert this change if we don't like it.